main.py

```diff
@@ -3,14 +3,31 @@
 # throughout this file
 import pygame
 
+from constants import SCREEN_HEIGHT, SCREEN_WIDTH
+
 
 def main():
-    try:
-        pygame.init()
-        print('Starting asteroids!')
-        pygame.quit()
-    except Exception as e:
-        print(f'Failed to initialize Pygame: {e}')
+    pygame.init()  # Initialize Pygame
```

Comment:
**Suggestion:** Add error handling for Pygame initialization.

**Existing Code:**
```Python
pygame.init()  # Initialize Pygame
```

**Improved Code:**
```Python
try:
    pygame.init()  # Initialize Pygame
except Exception as e:
    print(f'Failed to initialize Pygame: {e}')
    return
```

**Details:**
Implement error handling for Pygame initialization to ensure the game does not crash unexpectedly.